lib/render.ml

```diff
@@ -38,23 +28,19 @@ let create_control_points { c; rx; ry } =
 
 let draw_ellipse ctx ellipse =
   let start, curve_one, curve_two = create_control_points ellipse in
-  Cairo.save ctx.ctx;
   Cairo.move_to ctx.ctx start.x start.y;
   let x1, y1, x2, y2, x3, y3 = curve_one in
   Cairo.curve_to ctx.ctx x1 y1 x2 y2 x3 y3;
   let x1, y1, x2, y2, x3, y3 = curve_two in
   Cairo.curve_to ctx.ctx x1 y1 x2 y2 x3 y3;
-  Cairo.stroke ctx.ctx;
-  Cairo.restore ctx.ctx
+  Cairo.stroke ctx.ctx
 
 let draw_line ctx line =
-  save ();
-  let x1, y1 = denormalize line.a in
-  let x2, y2 = denormalize line.b in
+  let {x = x1; y = y1} = line.a in
+  let { x = x2; y = y2} = line.b in
   Cairo.move_to ctx.ctx x1 y1;
   Cairo.line_to ctx.ctx x2 y2;
-  Cairo.stroke ctx.ctx;
-  restore ()
+  Cairo.stroke ctx.ctx
```

Comment:
1. Can you run `dunt fmt` please?
2. Is restore () not necessary here?

Reply:
I'm not sure if it's necessary? I put it in there to be safe a while ago, but things seem to be rendering fine w/o it. I think `stroke` restores the original state? either way, removed it and formatted 👍🏻